### ISSUE TITLE
Feature request: Export a widget as json data, for rebuilding dynamic widget from the file

### ISSUE THREAD

**Issue by ronscoder** [dengyin2000/dynamic_widget#38]
Would it be possible, if we are able to export a widget to json data. Later we can import the json data and generate widget from it. 

This way, we can speed up building the json data.

**Comment by dengyin2000** [comment]
> Would it be possible, if we are able to export a widget to json data. Later we can import the json data and generate widget from it.
> 
> This way, we can speed up building the json data.

good idea,  we can build a tool for translate the widget dart code into dynamic widget json data.

**Comment by vicenterusso** [comment]
I like the idea. +1 from me

**Comment by MichealReed** [comment]
Wanted to add here that Rody Davis has created a really cool AST tool that does something similar. Maybe it can be adapted to work with this library? https://github.com/rodydavis/flutter_ast_core

**Comment by dengyin2000** [comment]
Hi guys, I create a DynamicWidgetJsonExportor widget for exporting your flutter code to json code, which you can build with dynamic widget package, please chekcout https://github.com/dengyin2000/dynamic_widget#how-to-write-the-json-code

**Comment by jwan16-ford** [comment]
> Hi guys, I create a DynamicWidgetJsonExportor widget for exporting your flutter code to json code, which you can build with dynamic widget package, please chekcout https://github.com/dengyin2000/dynamic_widget#how-to-write-the-json-code

我发现这个有bug，child里面有ListView的时候解析出来是null
